README.md

```diff
@@ -48,6 +56,41 @@ pacman -S base-devel git ncurses lib32-zlib gawk time unzip perl-xml-libxml \
  flex wget gettext quilt python2 openssl
 ```
 
+
+### Build-System-Structure  
+
+Where can I change something? What does these files do? 
+
+```
+:: Settings
+
+config.mk             - OpenWrt git repository, revision, select what package lists to build
+feeds.conf            - OpenWrt feeds.conf that is used for the package feeds during build
+
+configs/                  - further configurations
```

Comment:
"further configurations" replace by "snippets of the OpenWrt config-options (compare to call "make menuconfig" in openwrt/)" or something alike